proposals/nnnn-rc-radio-climate-parameter-update.md

```diff
@@ -0,0 +1,106 @@
+# Remote Control - Radio and Climate Parameter Update
+
+* Proposal: [SDL-NNNN](nnnn-rc-radio-climate-parameter-update.md)
+* Author: [Zhimin Yang](https://github.com/smartdevicelink/yang1070)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Android / RPC]
+
+## Introduction
+
+Currently, SDL only tells an application the total number of HD channels that are available for a given frequency via parameter `availableHDs` in `RadioControlData`. The application knows the maximum HD channel index is 7. However, it does not know exactly which HD sub-channel index has a valid broadcasting signal. This proposal adds a new read-only parameter `availableHdChannels`, which is a list of HD sub-channel indexes with HD radio broadcasting, to `RadioControlData` data structure. This proposal also adds a new parameter `availableHdChannelsAvailable` to `RadioControlCapabilities` data structure to indicate whether the vehicle supports the new parameter `availableHdChannels`.
+
+This proposal changes the minimum index of HD radio sub-channels from 1 to 0.
+
+In addition, we propose to add a new parameter `climateEnable` to `ClimateControlData` to allow an application to power on or off climate control.
+
+## Motivation
+
+SDL remote control shall allow a mobile application to know which HD sub-channel index has a signal so that the app can tune the HD radio to that sub-channel successfully.
+
+SDL remote control shall allow a mobile application to turn on/off the climate control if the vehicle supports it.
+
+
+
+## Proposed solution
+
+We propose to add a new parameter `availableHdChannels` to `RadioControlData` to allow an application to know what HD radio channel indexes are available.
```

Comment:
Change to:
**Add a new parameter** `availableHdChannels` to `RadioControlData` to allow an application to know what HD radio channel indexes are available.